services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/directory/decorator/DateAggregatingDecorator.java

```diff
@@ -88,8 +88,12 @@ public void prepare(ImmutableStack<Object> indices) {
               return aggregate;
             })
             .map(file -> new DiagnosisKeySigningDecorator(file, cryptoProvider))
-            .peek(currentDirectory::addWritable)
-            .forEach(aggregate -> aggregate.prepare(indices)));
+            .forEach(aggregate -> {
+              currentDirectory.addWritable(aggregate);
+              aggregate.prepare(indices);
+            }
+            )
```

Comment:
As both braces open on one line, also close them on the same.

```suggestion
            })
```